dlfcn_test.go

```diff
@@ -90,3 +94,33 @@ func Test_qsort(t *testing.T) {
 		}
 	}
 }
+
+
```

Comment:
One empty line is ifne (Probably you need gofmt)

Reply:
Applied, thanks.